TrackingTools/KalmanUpdators/interface/Chi2MeasurementEstimatorBase.h

```diff
@@ -18,12 +18,13 @@ class Chi2MeasurementEstimatorBase : public MeasurementEstimator {
    *  The errors of the trajectory state are multiplied by nSigma 
    *  to define acceptance of Plane and maximalLocalDisplacement.
    */
-  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3.) : 
-    theMaxChi2(maxChi2), theNSigma(nSigma) {}
+  explicit Chi2MeasurementEstimatorBase(double maxChi2, double nSigma = 3., float maxDisp=100.) : 
```

Comment:
I can set the constructor default to numeric_limits<float>::max
requiring a search window larger than 3m looks a bit exaggerated even in the muon chamber